### DISCUSSION TITLE
App is blank after clicking "add Apps"

### DISCUSSION THREAD

**Discussion by simonstewart-zindi** [envkey/envkey#32] [Q&A] [answered]
I'm running EnvKey 2.1.12 on a Mac.

When I click on "ADD APPS", I get a blank screen.

Possibly related, 50% of the time the loading screen when opening EnvKey never finishes and I need to force quit.

**Comment by danenania** [comment, marked as answer]
Hi @simonstewart-zindi, we're investigating. Could you please zip up any logs in the ~/.envkey/logs directory and either attach them here or send them to support@envkey.com?

Also, could you open the Developer Tools in the UI (View > Toggle Developer Tools), go to the Console tab, right click in the console and click 'Save As', and then attach or send that file too?

**Reply by danenania** [reply]
Also, does it help if you do you following:
- Close the EnvKey UI
- Run `envkey core stop` in a terminal
- Reopen the UI

**Reply by danenania** [reply]
@simonstewart-zindi This should now be fixed in the latest release. Please let me know if you see similar issues again!

**Reply by simonstewart-zindi** [reply]
I'll email the logs to you. 
The new app version didn't fix the loading issue unfortunately.

**Reply by danenania** [reply]
Should now be fixed in the latest version.

**Reply by simonstewart-zindi** [reply]
all sorted, thanks!
